rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] it would be better to make the rental add itself to the book and to the person like the below example so that will achieve the belongs-to association 
```ruby
 def initialize(date, book, person)
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```